tutorial/hextof_config.yaml

```diff
@@ -0,0 +1,151 @@
+core:
+  loader: flash
+  beamtime_id: 11019101
+  year: 2023
+  beamline: pg2
+  instrument: hextof
+  paths:
+    data_raw_dir: "/asap3/flash/gpfs/pg2/2023/data/11019101/raw/hdf/offline/fl1user3"
+    data_parquet_dir: "/home/agustsss/_projects/sed_dev/parquet/" #"/asap3/flash/gpfs/pg2/2023/data/11019101/processed/parquet"
```

Comment:
I think we can put a local default path here. Since we know it's on maxwell, "~/processed" should be work

Reply:
that is a bad thing to do. on maxwell you have very limited space on your home, and we should not pollute it with sed parquet files which a user would not be aware of.
There should better be no default and a definition of the paths in the notebook with an explaination.

Reply:
An idea would be to use OS managed temporary folders for the parquets generated when there is no other path. This would at least be more manageable "trash" for the users. Not sure how to implement this though

Reply:
I will push a new config soon, and was considering not putting anything in these fields. Populate them in the notebook instead.
Do we have a standard option of how to do this? to modify the settings in a notebook before passing them to generate an instance of SedProcessor?

Reply:
The only way I can think of is to import the config processor, load the config, and then add the key to the dict manually. 
Because when you call the SedProcessor, it immediately tries to call the loader and if the keys the loader requires don't exist, it'd just give an error..

Reply:
This exists perfectly fine. Load the config file as either user or system config, and provide the keys you want to modify in a dict as config

Reply:
E. G. https://github.com/OpenCOMPES/sed/blob/8141c9fd969dc3c87264d483f5125031fef2d3d5/tests/calibrator/test_energy.py#L42
Just for user_config insert your config file name.